tools/packaging/static-build/ovmf/build-ovmf.sh

```diff
@@ -36,9 +36,8 @@ info "Build ${ovmf_repo} version: ${ovmf_version}"
 
 build_root=$(mktemp -d)
 pushd $build_root
-git clone "${ovmf_repo}"
+git clone -b "${ovmf_version}" "${ovmf_repo}"
```

Comment:
Hi @fidencio ! Allow me suggest one thing: pass `--single-branch --depth 1` to speed up cloning. Here it finished in 7s (vs ~2 min).

The `git submodule update` below waste a good amount of time too to clone the BrotliCompress and openssl repositories. There should be a way to speed up their cloning too... do you know how?